src/chatdlg.cpp

```diff
@@ -58,29 +60,35 @@ CChatDlg::CChatDlg ( QWidget* parent ) :
     // set a placeholder text to make sure where to type the message in (#384)
     edtLocalInputText->setPlaceholderText ( tr ( "Type a message here" ) );
 
-
     // Menu  -------------------------------------------------------------------
     QMenuBar* pMenu     = new QMenuBar ( this );
     QMenu*    pEditMenu = new QMenu ( tr ( "&Edit" ), this );
 
-    pEditMenu->addAction ( tr ( "Cl&ear Chat History" ), this,
-        SLOT ( OnClearChatHistory() ), QKeySequence ( Qt::CTRL + Qt::Key_E ) );
+    pEditMenu->addAction ( tr ( "Cl&ear Chat History" ),
+                           this,
+                           SLOT ( OnClearChatHistory() ),
+                           QKeySequence ( Qt::CTRL + Qt::Key_E ) );
 
     pMenu->addMenu ( pEditMenu );
 
     // Now tell the layout about the menu
     layout()->setMenuBar ( pMenu );
 
-
     // Connections -------------------------------------------------------------
-    QObject::connect ( edtLocalInputText, &QLineEdit::textChanged,
-        this, &CChatDlg::OnLocalInputTextTextChanged );
-
-    QObject::connect ( butSend, &QPushButton::clicked,
-        this, &CChatDlg::OnSendText );
-
-    QObject::connect ( txvChatWindow, &QTextBrowser::anchorClicked,
-        this, &CChatDlg::OnAnchorClicked );
+    QObject::connect ( edtLocalInputText,
```

Comment:
I don't see why these got changed.  But I'd prefer the old formatting -- unfortunately there's an implicit pairing of the arguments that the formatter won't see.

Reply:
gets merged into 1 line with `ColumnLimit: 120`